client/src/Controller/Admin/IndexController.php

```diff
@@ -526,8 +536,54 @@ public function uploadOrgUsersAction(Request $request)
             }
         }
 
+        if ($processForm->isSubmitted() && $processForm->isValid()) {
+            /** Run the org CSV command as a background task */
+            $email = $this->tokenStorage->getToken()->getUser()->getUserIdentifier();
+            $this->dispatcher->addListener(KernelEvents::TERMINATE, function () use ($email) {
+                $application = new Application($this->kernel);
+                $input = new ArrayInput([
+                    'command' => 'digideps:process-org-csv',
+                    'email' => $email,
+                ]);
+
+                $output = new NullOutput();
+                $application->run($input, $output);
+            });
+
+
+            $this->addFlash("notice", "CSV import process started. Keep an eye on your emails for completion.");
+
+            return $this->redirect($this->generateUrl('admin_org_upload'));
+        }
+
+        /** S3 bucket information */
+        $bucket = $this->params->get('s3_sirius_bucket');
+        $paProReportFile = 'paProDeputyReport.csv';
```

Comment:
Worth making this use an environment var or constant so if naming changes at any point we can handle it easily.